pagai/engine/models/__init__.py

```diff
@@ -1 +1,3 @@
-from engine.models import train, predict
\ No newline at end of file
+from pagai.engine.models import train, predict
+
+SAVE_PATH = "pickle"
```

Comment:
should this be configurable ?

Reply:
I don't think so, it's internal and shouldn't be acceed by the suer